build/FrameworkPackage/CreateVpackFromNupkg.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+.SYNOPSIS
+Use this script to create a VPack that can be pushed to the OS.
+The input to this script should be a non-prerelease .nupkg containing the MUX Framework Packages.
+This script does not push the vpack, but gives the required command to do so.
+#>
+[CmdLetBinding()]
+Param(
+    [string]$NugetPackageInputPath,
+    [string]$VPackDirectoryOutputPath
+    )
+
+$nupkgFileNameWithoutExtension = (Get-Item $NugetPackageInputPath).Basename
+
+if(!$nupkgFileNameWithoutExtension.StartsWith("Microsoft.UI.Xaml.","CurrentCultureIgnoreCase"))
```

Comment:
Compare with Ordinal[IgnoreCase], CurrentCulture is almost always a red flag except for UI things.

Reply:
Good point. Fixing.